tests/page-objects/apdex/page.js

```diff
@@ -100,7 +112,15 @@ module.exports = class Page {
     }
   }
 
-  async loadAndAssertPage(page) {
+  async loadAndAssertPage(page, commonElements) {
+    if (!page) {
+      return;
+    }
+
+    if (page.relaunchApp) {
```

Comment:
New setting. 
Relaunching the app is now optional and we can specify it on any page.